docs-2.0/nebula-dashboard-ent/4.cluster-operator/1.overview.md

```diff
@@ -19,19 +21,35 @@ In this part, you can view the number of nodes as well as the number of running
 
 ### Alert
 
-In the **Alert** section, the system displays the five most recently triggered alert messages according to their severity level (`emergency`>`critical`>`warning`). For more information about alerts, see [Alerts](../9.alerts.md).
+In the **Alert** section, the system displays the five most recently triggered alert messages according to their severity level (`emergency`>`critical`>`warning`). 
+
+In the right upper corner, click **Alert Messages** to view alert messages. For more information about alerts, see [Alerts](../9.alerts.md).
 
 ### Information
 
-In this part, you can view the information of **Cluster name**, **Creation time**, **Creator**, and **Version**.
+In this part, you can view the information of **Cluster Name**, **Creation Time**, **Expiration Time**, **Creator**, and **Version**.
+
+- **Cluster Name**：The name of the cluster.
+- **Creation Time**：The time when the cluster is created.
+- **Expiration Time**：The time when the cluster license expires.
+
+  !!! enterpriseonly
+
+        The parameter **Expiration Time** is displayed only if the created or imported cluster is an Enterprise Edition.
```

Comment:
"Cluster is an Enterprise Edition"?

Consider:

The parameter **Expiration Time** is displayed only if the created or imported cluster is an Enterprise Edition cluster.
or
The parameter **Expiration Time** is displayed for only Enterprise Edition clusters.